lib/elixir/unicode/tokenizer.ex

```diff
@@ -1,6 +1,29 @@
 defmodule String.Tokenizer do
   @moduledoc false
 
+  ## Custom normalization definitions
+  #
+  # These codepoints will be normalized from => to, and their
+  # scriptset will be the union of both. If one of the two
+  # codepoints is Script='Common|Inherited', this means both
+  # codepoints can be used without unsafe script mixing;
+  # similarly, they are exempted from the Restricted list.
+  #
+  elixir_normalizations = %{
+    # NFKC-based normalizations for codepoints in use
+    'µ' => 'μ',
```

Comment:
Can we drop a comment with the codepoints on top? We can also use `?` to get the codepoint directly. Something like:

```suggestion
    # U+... => U+...
    ?µ => ?μ,
```